src/sqerl_pgsql_client.erl

```diff
@@ -110,9 +110,6 @@ execute_prepared({#prepared_statement{} = PrepStmt, Statements}, Parameters,
             epgsql:sync(Cn),
             {ok, Count};
         {ok, RowData} when is_list(RowData) ->
-            % query results, read-only, so no sync needed...?
-            % call it just in case, it shouldn't hurt
-            epgsql:sync(Cn),
```

Comment:
Based on the epgsql code, can we also remove the explicit sync in the case of a write since that should already be occurring as part of the extended query protocol flow?